frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -309,7 +309,7 @@ action.GitMachete.EnhancedGraphTable.branch-layout-write-failure=Writing new bra
 
 string.GitMachete.RediscoverSuggester.dialog.title=Git Machete: Rediscover Suggestion
 string.GitMachete.RediscoverSuggester.dialog.question=It looks that you have not modified your git machete file for a while. Would you like to rediscover the branch layout?
-
+string.GitMachete.RediscoverSuggester.backgroundable-check-task.title=Check if (re)discovery should be performed
```

Comment:
Sounds a bit too internal 🤔 what about smth like "Git Machete: Checking for repository update..."?